css/css-ui/appearance-button-list-box-select-001.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: appearance: button on list box select (multiple)</title>
+<link rel="help" href="https://drafts.csswg.org/css-ui-4/#appearance-switching">
+<meta name="assert" content="The button value acts like auto on a list box select.">
+<link rel="match" href="list-box-select-ref.html">
```

Comment:
That reference does not contain `appearance: auto`, and instead relies on the UA stylesheet having that value. Which in practice will be the case, but isn't yet (as far as I know) in any spec, and is a separate thing to test anyway. I suggest changing to a reference file that explicitly uses `appearance: auto`